### PR TITLE
feat(content-group-pictograms): introducing web-component version

### PR DESCRIPTION
**Pull request by guilhermelMoraes**
### Description

Introducing web-component version

### Changelog

**New**

- `content-group-pictograms.ts`;
- `content-group-pictograms.scss`;
- `content-group-pictograms.stories.js`

### CODE REVIEW

File: packages/web-components/src/components/content-group-pictograms/content-group-pictograms.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * @element dds-content-group-simple
+ * @slot media - The media content.
+ */
+
+@customElement(`${ddsPrefix}-content-group-pictograms`)
+class DDSContentGroupPictograms extends StableSelectorMixin(DDSContentGroup) {
+  // eslint-disable-next-line class-methods-use-this
+  protected _renderContent() {
+    return html`
+      <div class="${prefix}--content-group__children ${prefix}--content-group__col"></div>
```
Comment (asudoh) [inline packages/web-components/src/components/content-group-pictograms/content-group-pictograms.ts:31]: Seems that this `<div>` has nothing inside it, which made me wonder what this `<div>` is used for...?

**Review by asudoh — COMMENTED** [review]
Good progress @guilhermelMoraes!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @guilhermelMoraes!

**Comment by asudoh** [comment]
Update - Could you re-check the snapshot @guilhermelMoraes? Thanks!

**Comment by RobertaJHahn** [comment]
@guilhermelMoraes I am not sure why I see this PR on the zenhub board.  I usually don't see them so I am not sure what is different about this PR.  I do not close the PR's.  Do you know who should merge and close this?

**Comment by guilhermelMoraes** [comment]
@RobertaJHahn it's only waiting for any of the developers to approve the changes captured by Percy. It'll be closed and merged automatically once someone does that. The only thing I did was to connect this PR to the issue ticket.

**Pull request merged**